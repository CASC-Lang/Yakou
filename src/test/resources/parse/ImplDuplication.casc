class ImplDuplication

impl ImplDuplication

impl ImplDuplication