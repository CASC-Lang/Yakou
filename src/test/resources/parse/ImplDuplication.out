error: Duplicated type implementation for class ImplDuplication
--> ./ImplDuplication.casc:5:5
4 |
5 | impl ImplDuplication
         ^^^^^^^^^^^^^^^= hint: Consider remove this type implementation